src/pyOpenMS/tests/unittests/test_spectrum_experiment.py

```
@@ -0,0 +1,1086 @@
+    pyopenms.ChromatogramTools()
+    assert pyopenms.ChromatogramTools.convertChromatogramsToSpectra is not None
+    assert pyopenms.ChromatogramTools.convertSpectraToChromatograms is not None
+
+
+@report
+def testExperimentalSettings():
+    """
+    @tests: ExperimentalSettings
+     ExperimentalSettings.__init__
+    """
+    ff = pyopenms.ExperimentalSettings()
```

Ruff (0.14.10)

Local variable `ff` is assigned to but never used

Remove assignment to unused variable `ff`

(F841)